must-gather/collection-scripts/gather_cluster_logging_operator_resources

```diff
@@ -23,16 +25,18 @@ do
     get_env $pod $clo_folder $NAMESPACE "Dockerfile-*operator*"
 done
 
-oc -n $NAMESPACE get deployment cluster-logging-operator -o yaml > $clo_folder/deployment
+oc -n $NAMESPACE get deployment cluster-logging-operator -o yaml --cache-dir=${KUBECACHEDIR} > $clo_folder/deployment
 
 for csv in $(oc -n $NAMESPACE get csv -o name) ; do
-  oc -n $NAMESPACE get "${csv}" -o yaml > "${clo_folder}/${csv}.yaml"
+  mkdir -p "$(dirname ${clo_folder}/$csv)"
+  oc -n $NAMESPACE get "${csv}" -o yaml --cache-dir=${KUBECACHEDIR} > ${clo_folder}/"${csv}.yaml"
 done
 
+
 for r in "clusterlogging" "clusterlogforwarder" ; do
   data="$(oc -n $NAMESPACE get $r --ignore-not-found -o yaml)"
   if [ "$data" != "" ] ; then
-    echo "${data}" > "${clo_folder}/$r_instance.yaml"
+    echo "${data}" > "${clo_folder}/"$r"_instance.yaml"
```

Comment:
looks like an extraneous quote

Reply:
no, it is not. This is what i fixed.

before this change the file generated was `.yaml`
after this change the file generated is `clusterlogforwarder_instance.yaml`